sprocs/index.js

```diff
@@ -74,6 +74,9 @@ module.exports = {
             'config_select.sql',
             'users_select_or_insert.sql',
             'dump_to_csv.sql',
+            'calculate_assessment_question_stats_for_assessment.psql',
+            'calculate_assessment_question_stats.psql',
+            'calculate_stats_for_instance_questions.psql'
```

Comment:
For filenames in `sprocs/` I'm trying to use a naming convention where we start with the name of the table that's being operated on. For example, `sprocs/instance_questions_grade.sql` performs the `grade` operation on `instance_questions`. Given this, could we rename this file to `instance_questions_update_stats.sql` or something like that? The above two files could be `assessment_questions_update_stats.sql` and `assessment_questions_update_stats_for_assessment.sql`.